datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -413,4 +416,14 @@ private String getAccountIdFromCrn(String userCrn) {
     private DetailedEnvironmentResponse getEnvironment(String environmentName) {
         return environmentClientService.getByName(environmentName);
     }
+
+    private List<SdxCluster> filterOutClustersWithDeletedStatus(List<SdxCluster> sdxClusters) {
```

Comment:
This will just hide the original problem, I think we should rather fix the source of the problem.
@sodre90 has on his branch an implementation that will make the Status change and Deleted timestamp addition so this won't happen again.

For fixing the already stuck clusters we should create an SQL script.
I see there are also some additional unit tests, those can be useful.

Talk with @sodre90 please.